src/Umbraco.Web/Unversion/UnversionComposer.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Umbraco.Core;
+using Umbraco.Core.Composing;
+
+namespace Umbraco.Web.Unversion
+{
+    public class UnversionComposer : ICoreComposer
+    {
+        public void Compose(Composition composition)
+        {
+            composition.Register<IUnversionConfig, UnversionConfig>();
```

Comment:
IUnversionConfig and IUnversionService should be part of core, they don't rely on anything in web and should be able to run outside of web if required. Probably should be registered how our other services and config are registered.